### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by pierreduchemin**
The idea here is just to make it easier to compile in 2019.

### CODE REVIEW

**Comment by btkelly** [comment]
@pierreduchemin Thanks for the PR! Out of curiosity, why did you remove the Gnag checker plugin? We keep it in there to enforce styling and good practices.

**Comment by pierreduchemin** [comment]
@btkelly I indeed removed it since it was not required in my context and was blocking me on first build. I can try to put it back though.

**Comment by btkelly** [comment]
O right, that was an issue with how Travis exposes variables on PR builds that are not from contributors with write access to the repo. I'll pull this down and take a look this afternoon, you can ignore the Travis failure. Thanks again for the contributions!

File: gandalf/build.gradle [hunk 1]
```diff
@@ -63,40 +64,12 @@ gnag {
...
-
-    developerId = 'btkelly'
-    developerName = 'Bryan Kelly'
-    developerEmail = 'bobbake4@gmail.com'
-}
-
-apply from: 'https://raw.githubusercontent.com/btkelly/JCenter/f988f0e106be89c876e038894b9892f9d1d0b643/bintrayv1.gradle'
-apply from: 'https://raw.githubusercontent.com/btkelly/JCenter/master/installv1.gradle'
+    testImplementation 'junit:junit:4.12'
+    testImplementation 'org.mockito:mockito-core:2.7.13'
```
Comment (btkelly) [inline gandalf/build.gradle]: Can we update this to `3.0.0` to fix support for newer JDK versions?

**Comment by pierreduchemin** [comment]
> add `android:usesCleartextTraffic="true"` to the Example application's `AndroidManifest.xml`. Currently the example app does not function properly on newer Android versions but with that change it should be set.

What kind of issue are you running into? I'm testing on Android 8.1.0 and it doesn't looks like there's any problem.
It make sense though since we are on a mock.

**Comment by btkelly** [comment]
I'm testing on a Pixel 2 with Android 9 and because the device forces `https` traffic only the localhost mock server calls always fail. That attribute just tells Android to remove the `https` requirement.

**Comment by pierreduchemin** [comment]
That's why: "Attribute usesCleartextTraffic is only used in API level 23 and higher (current min is 16)"

File: gandalf/build.gradle (hunk 1 above)
Comment (pierreduchemin) [reply gandalf/build.gradle]: Done in https://github.com/btkelly/gandalf/pull/87/commits/6acaec4225df7a82e5fff584f4280d46d30d5eb1

**Comment by pierreduchemin** [comment]
Hello,
anything else needs to be done here to get this approved?

**Comment by btkelly** [comment]
Nope, I'll merge this and release a new version shortly 🥇 thanks again!

**Pull request merged**